### PR TITLE
diagnostic: Update whitelisted dylib used by Symantec

### PR DESCRIPTION
**Pull request by wang-boyu**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
  * This is to fix a `brew doctor` warning about unbrewed dylibs used by Symantec Endpoint Protection
  * Previously some dylibs and static libs have been added into white_list in [this pull request](https://github.com/Homebrew/brew/pull/1438)
- [x] ~~Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).~~
- [x] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

File: Library/Homebrew/diagnostic.rb [hunk 1]
```diff
@@ -166,6 +166,7 @@ def check_for_stray_dylibs
           "libUFSDExtFS.dylib", # Paragon ExtFS
           "libecomlodr.dylib", # Symantec Endpoint Protection
           "libsymsea.*.dylib", # Symantec Endpoint Protection
```
Comment (MikeMcQuaid) [inline Library/Homebrew/diagnostic.rb]: Could this line be edited instead to be `"libsymsea*.dylib"?

Comment (wang-boyu) [inline Library/Homebrew/diagnostic.rb]: updated as suggested!

Comment (MikeMcQuaid) [inline Library/Homebrew/diagnostic.rb]: @wang-boyu Last thing: just wanted to check it still worked for you.

Comment (wang-boyu) [inline Library/Homebrew/diagnostic.rb]: @MikeMcQuaid Yes still working for me - no more warning about unbrewed dylibs.

**Comment by MikeMcQuaid** [comment]
@wang-boyu Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by wang-boyu** [comment]
@MikeMcQuaid lol thanks!!

**Pull request merged**